assets/js/components/UserMenu/index.js

```diff
@@ -223,58 +232,69 @@ export default function UserMenu() {
 					aria-haspopup="menu"
 					aria-expanded={ menuOpen }
 					aria-controls="user-menu"
-					aria-label={ __( 'Account', 'google-site-kit' ) }
+					aria-label={
+						isAutoCreatingCustomDimensionsForAudience
+							? ''
```

Comment:
I'd suggest `undefined` instead of an empty string, to avoid setting the attribute at all...

```suggestion
							? undefined
```